api/src/routes/homepage.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * This module defines the endpoints for the "/homepage" route and exports the corresponding Router.
+ */
+
+const homepageRouter = require('express').Router();
+
+const homepageController = require('../controllers/homepage');
+
+homepageRouter.post('/:team_id', homepageController.createOrUpdateHomepage);
+
+homepageRouter.get('/:team_id', homepageController.getHomepage);
```

Comment:
Also, remove the new lines between the relative imports for the same consistency reason. (basically remove newlines on 8, 10, etc. to make sure all relative imports are bunched together)